cpp/daal/src/externals/service_memory.h

```diff
@@ -138,6 +138,17 @@ void service_memset_seq(T * const ptr, const T value, const size_t num)
     }
 }
 
+template <typename T, CpuType cpu>
+void service_memset_ser(T * const ptr, const T startValue, const size_t num)
```

Comment:
Not obvious meaning of function name. Maybe, `service_memset_range`?

Reply:
I went with `service_memset_incrementing`, do you agree?